blog/_posts/2016-02-20-announcing-open-spending-alpha-version-release.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Introducing OpenSpending Next Alpha release
+authors: 
+- Victoria Vlad, Cecile Le Guen
+---
+
+Over the past few months, the team at Open Spending has been working hard to deliver a brand new version of OpenSpending and we’re now excited to announce the Alpha version release of Open Spending Next. The new version is targeted to all users who have an interest in advancing the openness of budget and spending data. 
+
+
+Whether you are a non-technical person, or a data professional, whether you're journalist or an administrator, Open Spending Next is getting ready for your use!
```

Comment:
I think we should emphasize what are the benefits of the new version vs the current version, i.e. what have we been working so hard on? (this same comment goes for the landing page).